kontroller/qml/ServerSettingsPage.qml

```diff
@@ -152,8 +152,8 @@ Dialog {
                 anchors.leftMargin: Theme.horizontalPageMargin
                 anchors.rightMargin: Theme.horizontalPageMargin
                 inputMethodHints: Qt.ImhNoAutoUppercase
-                label:qsTr("Login");
-                placeholderText: qsTr("Login");
+                label:qsTr("Username");
```

Comment:
Yes?

Reply:
Indeed, this is what kodi displays in the settings page.